src/Aspire.ProjectTemplates/templates/aspire-starter/.template.config/localize/templatestrings.fr.json

```diff
@@ -5,28 +5,28 @@
   "symbols/Framework/description": "Framework cible du projet.",
   "symbols/Framework/choices/net8.0/description": "Cible net8.0",
   "symbols/Framework/choices/net9.0/description": "Cible net9.0",
-  "symbols/AspireVersionCli/description": "The version of .NET Aspire to use.",
-  "symbols/AspireVersionCli/displayName": ".NET Aspire version",
+  "symbols/AspireVersionCli/description": "La version de .NET Aspire à utiliser.",
+  "symbols/AspireVersionCli/displayName": "Version .NET Aspire",
   "symbols/AspireVersionCli/choices/9.1/displayName": "9.1",
-  "symbols/AspireVersionCli/choices/9.1/description": "Chooses .NET Aspire 9.1",
+  "symbols/AspireVersionCli/choices/9.1/description": "Choisit .NET Aspire 9.1",
   "symbols/AspireVersionCli/choices/9.0/displayName": "9.0",
-  "symbols/AspireVersionCli/choices/9.0/description": "Chooses .NET Aspire 9.0",
+  "symbols/AspireVersionCli/choices/9.0/description": "Choisis .NET Aspire 9.0",
   "symbols/AspireVersionCli/choices/8.2/displayName": "8.2",
-  "symbols/AspireVersionCli/choices/8.2/description": "Chooses .NET Aspire 8.2",
-  "symbols/AspireVersion/description": "The version of .NET Aspire to use.",
-  "symbols/AspireVersion/displayName": ".NET Aspire version",
+  "symbols/AspireVersionCli/choices/8.2/description": "Choisit .NET Correctement 8.2 (valide uniquement quand '--framework net8.0' est spécifié)",
```

Comment:
@adamint good idea when you review to stop check these to make sure they didn't translate `Aspire`.  Ill open a  bug.

Not sure about JSON but in RESX you can mark words as to not translate. @DamianEdwards maybe you know as this is templates

Reply:
OK to merge though for now

Reply:
https://ceapex.visualstudio.com/CEINTL/_workitems/create/Feedback?templateId=532ad3a8-b25f-4217-a028-979ac4c34340&ownerId=354effea-8ab7-482a-946d-358eae314712

Reply:
@danmoseley apologies, I didn't notice that aspire was even translated here. I'll check in the future - as a product name, I wasn't expecting them to try to localize it. Should I revert this commit?

Reply:
@adamint no apologies required, just something that commonly happens. I merged since I think it is a net improvement, but either way seems fine.